assets/blocks/course-outline/data.js

```diff
@@ -71,26 +71,51 @@ export const syncStructureToBlocks = ( structure, blocks ) => {
 	} );
 };
 
+/**
+ * Predicate method to compare a block with a course structure item, using different strategies.
+ *
+ * @param {Object[]}                                    blocks        Block.
+ * @param {Array.<(CourseLessonData|CourseModuleData)>} structureItem Structure item.
```

Comment:
Thanks for updating! :) Should we update the param names in jsdoc as well to reflect the updated params of the fuction?

Reply:
Sorry for the backing and forwarding
Update here https://github.com/Automattic/sensei/pull/5363/commits/2898e5b85633486456102f26b34177999199edbf

I remove the manual Currying to use it from lodash because I didn't find a way to describe it using JSDocs.